src/patients/new/NewPatientForm.tsx

```diff
@@ -37,6 +38,9 @@ const NewPatientForm = (props: Props) => {
   })
 
   const onSaveButtonClick = async () => {
+    if (!patient.givenName && !patient.familyName) {
+      return setError("No patient name entered!")
```

Comment:
This string needs to support internationalization. 

To do that you can add a key and value to `locales/en-US/translation.json`. I think a key in the `patient` object called `patientNameRequired` or something like that would work.